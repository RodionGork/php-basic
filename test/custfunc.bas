x = 13
def fnt(x) = x * (x+1) / 2
print "triangle number for 10 is", fnt(10)
def fns(y) = fnt(y) + fnt(y - 1)
print "square of 10 is sum of two triangles:", fns(10)
print "And X should remain thirteen:", x