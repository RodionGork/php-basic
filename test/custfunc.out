triangle number for 10 is 55
square of 10 is sum of two triangles: 100
And X should remain thirteen: 13
